### DISCUSSION TITLE
Compare 2 files in a project feature request

### DISCUSSION THREAD

**Discussion by NioForce** [phcode-dev/phoenix#1808] [Ideas]
Hello Team,
Excellent code editor Phoenix is.
Can you please add a feature so that 2 files in a project or from different project be compared?
Thank you.

**Comment by abose** [comment]
Git support will come in the September release of phcode. Git diff viewer will come with it.

But for diffing between any files, let me see if it can be done with the git diff framework we have. Tracked as https://github.com/phcode-dev/phoenix/issues/1062
